client/components/login/LoginForm.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <v-sheet tile elevation="0" color="white">
+    <v-container class="py-16">
+      <v-row>
+        <v-col cols="6" class="mx-auto">
+          <h2 style="text-align: center; margin-bottom: 1rem">Login</h2>
```

Comment:
Try to use the vuetify in-built classes where possible.
```suggestion
          <h2 class="text-center mb-4">Login</h2>
```